site/layouts/partials/home/masthead.html

```diff
@@ -2,7 +2,7 @@
   <div class="container-xxl bd-gutter">
     <div class="col-md-8 mx-auto text-center">
       <a class="d-flex flex-column flex-lg-row justify-content-center align-items-center mb-4 text-dark lh-sm text-decoration-none" href="https://www.herodevs.com/support/nes-bootstrap?utm_source=Bootstrap_site&utm_medium=Banner&utm_campaign=v3and4_eol" rel="noopener" target="_blank">
-        <span class="d-sm-inline-flex align-items-center gap-1 py-2 px-3 me-2 mb-2 mb-lg-0 rounded-5 masthead-notice">
+        <span class="pb-2 px-3 me-2 mb-2 mb-lg-0 rounded-5 masthead-notice">
```

Comment:
```suggestion
        <span class="py-2 px-3 me-2 mb-2 mb-lg-0 rounded-5 masthead-notice">
```
And you can remove the `padding-top` from `.masthead-notice`